prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -546,6 +551,7 @@ def prompt_processor() -> Any:
                             usage_kwargs=usage_kwargs,
                         )
                         challenge = challenge_plugin["entrypoint_cls"](
+                            llm=llm,
```

Comment:
Do we have a corresponding PR to handle this new param for challenge @Deepak-Kesavan ?

Reply:
Got it - https://github.com/Zipstack/unstract-cloud/pull/119